examples/custom/autocomplete/autocomplete_input.py

```diff
@@ -0,0 +1,22 @@
+from bokeh.core.properties import List, String, Int, Bool
+from bokeh.models.widgets import TextInput
+
+class AutocompleteInput(TextInput):
+    """ Single-line input widget with auto-completion. """
+
+    __implementation__ = "autocomplete_input.coffee"
+    __css__ = ["https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.css"]

```

Comment:
This seems unnecessary as you use built-in jquery-ui in the implementation. Note that we use prefixed CSS classes, so there will be unwanted interactions between those two versions or this include will be a no-op.